addon/-private/request.js

```diff
@@ -3,7 +3,7 @@ import { Promise } from 'rsvp';
 
 export default function (url, options) {
   return new Promise((resolve, reject) => {
-    let xhr = options.xhr ? options.xhr() : new XMLHttpRequest();
+    let xhr = options.xhr();
```

Comment:
As it's now required, I suggest we add `xhr` as the first argument of `request()` function instead of expecting it in the `options` object

Reply:
Done now.